### ISSUE TITLE
Easy way to add new accessory

### ISSUE THREAD

**Issue by mcruscher** [madchicken/homebridge-zigbee-nt#46]
Hello.

I want to add new accessory (Mi Smoke Detector - Honeywell) and tried to follow the described steps in the wiki.
These stept are hard to follow. Can you outline the more detailed for a newbie?
Or perhaps add a function in the UI?

Greatings.

**Comment by tr1ng0** [comment]
Hey @mcruscher not sure if u already resolved this.
Did you follow the intructions as followed? https://github.com/itsmepetrov/homebridge-zigbee/blob/master/docs/PairingDevices.md

**Comment by madchicken** [comment]
@mcruscher can you take a look [here](https://github.com/madchicken/homebridge-zigbee-nt/wiki/How-to-support-new-devices)?
